src/brs/Constants.java

```diff
@@ -68,6 +68,10 @@
   public static final int MAX_AUTOMATED_TRANSACTION_NAME_LENGTH = 30;
   public static final int MAX_AUTOMATED_TRANSACTION_DESCRIPTION_LENGTH = 1000;
 
+
+  public static final int PEER_DEFAULT_PORT = 8123;
```

Comment:
well, seems that are some legacy ports. Over the years 8125 and 6786 has been established for main and test net. :shrug: - But I agree, that these should be in constants

Reply:
8123 is what the peers talk to each other on. 7123 is what they use for p2p on test net. Live net is configurable but test net is hard coded. I assume that's so live net nodes can identify the test net easily and not accept blocks from it.